src/task3/main.java

```diff
@@ -0,0 +1,32 @@
+package task3;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class main {
+    public static void main(String[] args) {
+        List<Integer> list = List.of(12, 32, 12, 3213, 213, 12, 32);
+        System.out.println(findUnique(list));
+    }
+
+    public static <T> List<T> findUnique(List<T> list) {
+        List<T> newList = new ArrayList<>();
```

Comment:
It seems, that you're implementation of this method is incorrect. Please check the task description carefully.

Reply:
Sorry, but this implementation is still not really correct. Why do you have 2 loops in your implementation? One loop is more than enough. Also, you can use HashSet to solve this method in 1 line.